python/py-pkgutil_resolve_name/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pkgutil_resolve_name
+version             1.3.10
+revision            0
+categories-append   devel
+license             MIT
+supported_archs     noarch
+
+python.versions     37 38
+
+maintainers         nomaintainer
+
+description         A backport of Python 3.9’s pkgutil.resolve_name.
+long_description    {*}${description}
+
+homepage            https://github.com/graingert/pkgutil-resolve-name
+
+checksums           rmd160  d63e02f9beb87d70e1f9c95076176233eac639cd \
+                    sha256  357d6c9e6a755653cfd78893817c0853af365dd51ec97f3d358a819373bbd174 \
+                    size    5054
+
+if {${name} ne ${subport}} {
+    livecheck.type      none
+}
```

Comment:
the `livecheck.type none` is the default set by the `python` PG if there are no `version` overrides in the Portfile. So that file is redundant here and therefore the whole `if`-block can be removed.

Reply:
Done. Although I had to keep the `if`-block to reinplace `pyproject.toml` – see previous issue.